code/modules/client/preferences.dm

```diff
@@ -686,23 +688,26 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			dat += "<tr><td colspan=4><hr></td></tr>"
 			dat += "<tr><td><b>Name</b></td>"
-			dat += "<td><b>Cost</b></td>"
+			if(LC.category != "Donator" || !CONFIG_GET(flag/donator_items))
+				dat += "<td><b>Cost</b></td>"
```

Comment:
Players of servers that haven't set up patron (or any other equivalent) won't be able to access the donator gear category panel and, therefore, donator gears, so we can skip config check here.

```suggestion
			if(LC.category != "Donator")
				dat += "<td><b>Cost</b></td>"
```